### PR TITLE
Add org.gabmus.whatip

### PR DESCRIPTION
**Pull request by GabMus**

### CODE REVIEW

**Comment by GabMus** [comment]
bot, build org.gabmus.whatip

**Comment by GabMus** [comment]
oops, forgot to add an hash

**Comment by GabMus** [comment]
bot, build org.gabmus.whatip

**Comment by GabMus** [comment]
bot, build org.gabmus.whatip

**Comment by GabMus** [comment]
Alright this is a weird error:

```
flatpak-builder: Downloading whatipFailed to download sources: module whatip: No checksum specified for archive source whatip-0.1.tar.gz
```

But the hash is there. Any help? Thanks

File: org.gabmus.whatip.json [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "finish-args" : [
+    	"--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-run/dconf",
```
Comment (nedrichards) [inline org.gabmus.whatip.json]: these dconf permissions are now built in unless you need something special from the system integration and should be removed.

File: org.gabmus.whatip.json [hunk 2]
```diff
@@ -0,0 +1,339 @@
+{
+    "app-id" : "org.gabmus.whatip",
+    "command" : "whatip",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "finish-args" : [
+    	"--device=dri",
```
Comment (nedrichards) [inline org.gabmus.whatip.json]: nitpick on the indentation

**Comment by nedrichards** [comment]
@GabMus there was a build system hiccup, fixed now. App seems to work well, although I don't see anything displayed on the LAN panel, in which case the empty state of the panel looks kinda ugly. I assume it's supposed to display any kind of bonjour service? If so, I have plenty of them!

File: org.gabmus.whatip.json [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+                    ]
+                }
+            ]
+        },
+        {
+            "name" : "python-urllib3",
+            "buildsystem" : "simple",
+            "build-options" : {
+                "arch" : {
+                    "i386" : {
```
Comment (bilelmoussaoui) [inline org.gabmus.whatip.json]: You can remove all the i386 specific things.

**Comment by GabMus** [comment]
bot, build org.gabmus.whatip

**Comment by GabMus** [comment]
any news?

**Comment by bilelmoussaoui** [comment]
bot, build org.gabmus.whatip

**Comment by bilelmoussaoui** [comment]
The app crashes whenever I try to run it 
```
  File "/app/bin/whatip", line 68, in <module>
    __main__.main()
  File "/app/lib/python3.7/site-packages/whatip/__main__.py", line 158, in main
    application = GApplication()
  File "/app/lib/python3.7/site-packages/whatip/__main__.py", line 36, in __init__
    self.window = AppWindow()
  File "/app/lib/python3.7/site-packages/whatip/app_window.py", line 23, in __init__
    self.headerbar = GHeaderbar(self.main_stack)
  File "/app/lib/python3.7/site-packages/whatip/headerbar.py", line 61, in __init__
    self.on_refresh_btn_clicked(sync=True)
  File "/app/lib/python3.7/site-packages/whatip/headerbar.py", line 83, in on_refresh_btn_clicked
    self.main_stack.port_listbox.refresh()
  File "/app/lib/python3.7/site-packages/whatip/port_listbox.py", line 39, in refresh
    ports[port]['type']
  File "/app/lib/python3.7/site-packages/whatip/port_listbox_row.py", line 71, in __init__
    )[self.port_type.lower()]
KeyError: 'udp'
```

**Comment by GabMus** [comment]
bot, build org.gabmus.whatip

**Pull request closed without merging**